load-generator/src/main/java/com/amazon/opentelemetry/load/generator/emitter/MetricEmitter.java

```diff
@@ -32,8 +31,8 @@ public abstract class MetricEmitter implements Emitter {
 
   @Override
   public void start(Runnable emitter) {
-    log.info("Generating metrics at a rate of(ms) : {}" , this.param.getObservationInterval());
-    scheduler.scheduleAtFixedRate(emitter, 0, this.param.getObservationInterval(), TimeUnit.MILLISECONDS);
+    log.info("Generating metrics at a rate of(ms) : {}" , FLUSH_INTERVAL);
+    scheduler.scheduleAtFixedRate(emitter, 0, FLUSH_INTERVAL, TimeUnit.MILLISECONDS);
```

Comment:
this simplifies things a bit. Should we add an warning/raise an exception in case the emitter function is not able to complete in less than 1s? Otherwise the ingestion rate will not be correct right?

Reply:
I am unsure how to implement this. `scheduleAtFixedRate` only throws these three exceptions:

> RejectedExecutionException – if the task cannot be scheduled for execution
> NullPointerException – if command or unit is null
> IllegalArgumentException – if period less than or equal to zero